### ISSUE TITLE
Cant download VODS

### ISSUE THREAD

**Issue by xentumm** [TwitchRecover/TwitchRecover#416]
When I paste twitchtracker stream link app goes down itself what can i do its so important to me

**Comment by leonid-dalin** [comment]
Having similar issues here. The application crashes as soon as I enter the twitchtracker link.

**Comment by Bananamax** [comment]
Yes same for me. It worked flawlessly though and sth. changed. Probably a Windows 11 update?! What now

**Comment by yassindeb** [comment]
could you reply with the twitchtracker link?
